### ISSUE TITLE
del-cli not found when installing

### ISSUE THREAD

**Issue by rockers7414** [sequelize/umzug#210]
As title, any idea?

Log:
```
npm install sequelize/umzug                         

> umzug@3.0.0-beta.2 prepack /home/rockers7414/.npm/_cacache/tmp/git-clone-8e0f4b89
> npm run build


> umzug@3.0.0-beta.2 build /home/rockers7414/.npm/_cacache/tmp/git-clone-8e0f4b89
> del-cli lib && tsc

sh: 1: del-cli: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! umzug@3.0.0-beta.2 build: `del-cli lib && tsc`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the umzug@3.0.0-beta.2 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/rockers7414/.npm/_logs/2020-04-06T05_25_06_091Z-debug.log
npm ERR! premature close

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/rockers7414/.npm/_logs/2020-04-06T05_25_06_118Z-debug.log
```

**Comment by rockers7414** [comment]
After I rename `prepack` script to `prepare`, I can install it successfully. BTW, it also needs to revise the `main` to `lib/src/index.js`.

I'm not sure this is the correct solution.

**Comment by papb** [comment]
Thanks for the report! Weird though, I can't reproduce it. What is this `npm install sequelize/umzug` syntax? I would expect `npm install umzug@beta` instead.

> BTW, it also needs to revise the `main` to `lib/src/index.js`.

Ouch, this is very true, thanks for spotting that :man_facepalming:

**Comment by papb** [comment]
Also I think the exact name `prepack` is important for `np` to publish the package correctly...

**Comment by papb** [comment]
Fixed by [`umzug@3.0.0-beta.3`](https://github.com/sequelize/umzug/releases/tag/v3.0.0-beta.3)
